dvc/progress.py

```diff
@@ -85,6 +86,14 @@ def _bar(self, target_name, current, total):
 
         return num + bar + percent + target_name
 
+    def _print(self, *args, **kwargs):
```

Comment:
It still does all the `Progress` operations, I'm just not printintg the result, turning them into useless process, but it was the most straight forward (low-impact) solution I could find.